### ISSUE TITLE
Cockatrice plugin interface

### ISSUE THREAD

**Issue by Arteneko** [Cockatrice/Cockatrice#3885]
Does cockatrice provide a plugin interface to extend the software?

**Comment by ebbit1q** [comment]
No, but the entire source code is extendable. What do you want out of a plugin interface anyway? Simply having the source code documented better?

**Comment by Arteneko** [comment]
No, but rather, being able to use cockatrice to manage card inventories, and not just decks; and having the Scryfall API integrated to search cards in different languages

**Comment by ZeldaZach** [comment]
Welome @Arteneko! We're always accepting of pull requests here for awesome features that would increase the user experience of our program. If you'd ever like to chat with the dev team, we have a Discord and Gitter where we discuss everything
